SashaZaitsev/HW6/06.Collections.py

```diff
@@ -0,0 +1,20 @@
+# Сollections
+for i in range(1, 10):
+
+    if i % 2 == 0:
+        print(i, end = ' ')
+        print("Числа парні")
+    elif i % 2 != 0 and i % 3 == 0:
+        print(i, end = ' ')
+        print("Непарні, що діляться на три")
+    else:
+        print(i, end = ' ')
+        print("Числа, що не діляться на три та два")
+        
+while True:
```

Comment:
good job
look at this realization
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)